scripts/deploy-token-paymaster-mumbai.ts

```diff
@@ -5,6 +5,7 @@ async function main() {
   const provider = ethers.provider;
 
   const accounts = await ethers.getSigners();
+  const ZERO_ADDRESS_ROUTER = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
```

Comment:
Don't see this variable being used in this file.

It's name should just be NATIVE_ADDRESS

Reply:
fixed